HW5/HW5.py

```diff
@@ -0,0 +1,70 @@
+# 1 TASK---------------------------------------------
+
```

Comment:
look at this implementation

list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)

Reply:
Ok, I`ll check